### ISSUE TITLE
Stops connecting to RF Explorer

### ISSUE THREAD

**Issue by ShanethPatterson** [berkon/wireless-microphone-analyzer#12]
Hi!
I have been using Wireless Microphone Analyzer since fall last year. It has been excellent for my usage! 
However, after updating to 1.7.3, the RF Explorer no longer shows scan results from the RF Explorer, but it does show the USB port with a checkmark. It also does not show the RF range the explorer is set to.
Downgrading back to 1.7.1 does not seem to fix it.
Thank you!

**Comment by berkon** [comment]
Hi @ShanethPatterson, which version did you use before? I'm asking because V1.7.1 was published on 19th January. Thus you must have had an older version, when you used it in fall. Would be good if you could find that out, then I can check what changes were done since then.

**Comment by ShanethPatterson** [comment]
It must have been 1.6.1!

**Comment by berkon** [comment]
Please try the following:

- You said you downgraded to 1.7.1. Can you please try to downgrade to 1.6.1 and check if it works again.
- Press <CTRL><ALT><SHIFT><i> and let me know if there are any error messages on the console. Just post the output here
- Let me know which platform are you using

**Comment by ShanethPatterson** [comment]
<img width="1312" alt="Messages Image(3274187056)" src="https://user-images.githubusercontent.com/48448870/117475245-cd6e2600-af29-11eb-8d12-673c3b0270c6.png">

**Comment by ShanethPatterson** [comment]
I am running mac os 10.14.6 Mojave on a 2015 Macbook Pro Retina 15 inch

**Comment by berkon** [comment]
Ah, ok. I think I had that issue as well sometimes. But I didn't find a reason why it happend. It doesn't seem to be an issue with the software. In my case, as far as I remember, I disconnected the device and made sure that it is turned off (I found out that sometimes the firmware hangs). Also wait a few seconds before you connect it again (also the USB connectors sometimes go crazy). Also stop the Microphone Analyzer Tool. Then connect the device again, wait a few seconds for USB to discover it and then start the tool. If that doesn't work, I'd suggest to downgrade to 1.6.1, just to check if that helps. If yes, then there indeed might be an issue in the software which I can look after.
